DEVELOPMENT.md

```diff
@@ -18,8 +18,8 @@ Prerequisites:
     * vscode-just
     * WSL
 * PowerShell 7
-* Rust development tools for version listed in `rust-toolchain.toml`
-* `rustup toolchain install nightly`
+* Rust development tools for version listed in `rust-toolchain.toml` (currently 1.87)
```

Comment:
This information is bound to go out of sync, do not re-state data from other files. Applies to this line and elsewhere in this file.